lib/install-panel.js

```diff
@@ -207,12 +206,11 @@ export default class InstallPanel {
     this.refs.searchMessage.textContent = `Searching ${this.searchType} for \u201C${query}\u201D\u2026`
     this.refs.searchMessage.style.display = ''
 
-    const opts = {}
-    opts[this.searchType] = true
-    opts['sortBy'] = 'downloads'
+    const options = {sort: {downloads: 'desc'}}
```

Comment:
Should just be `const options = {sort: 'downloads'}`.  The atom.io API is documented at https://flight-manual.atom.io/atom-server-side-apis/sections/atom-package-server-api/.  Though it seems like the `direction` parameter is a bit broken for downloads because I can't get it to sort ascending...luckily we sort descending :grinning:.

Reply:
`const options = {sort: 'downloads'}`
![screen shot 2018-01-05 at 15 20 09](https://user-images.githubusercontent.com/1017132/34610925-fe85759a-f22b-11e7-889f-a576638a2053.png)

`const options = {sort: {downloads: 'desc'}}`
![screen shot 2018-01-05 at 15 19 11](https://user-images.githubusercontent.com/1017132/34610928-025721d2-f22c-11e7-8d05-bcdd2e009ca1.png)

Reply:
Considering sorting was not done before, I am guessing it can be straight omitted and perhaps the reason why it works is because it has no effect (unconfirmed).

Reply:
Yes, it was because of the garbage value in sort field why it seemed to work.

However, when properly sorting on serverside vs not sorting at all, we are getting way different results. The default, unsorted search seems to give more relevant results compared to what is popular, but less relevant. I will roll back the sorting part from last night.